### PR TITLE
fix(products): enable flip image feature

### PR DESCRIPTION
**Pull request by virginiaritonga**
homepage:

https://user-images.githubusercontent.com/43097333/164198452-37afe582-9fec-4b4e-b1c1-5cfb8867c376.mov

PLP:

https://user-images.githubusercontent.com/43097333/164198559-b7b98df5-3d44-4783-a530-ff64ab0888b1.mov

PDP:

https://user-images.githubusercontent.com/43097333/164198665-56715717-bbb4-4c0e-b72d-763fcfee0994.mov

Cart:

https://user-images.githubusercontent.com/43097333/164199340-1bf81e01-a77c-4034-ab28-86ddea7c9616.mov

### CODE REVIEW

File: components/ProductsComponent/ProductRecomendation.tsx [hunk 1]
```diff
@@ -85,6 +85,7 @@ const ProductRecomendation: FC<Iprops> = ({
         filter={{ openOrderScheduled: false, published: true }}
         fullPath={`product/{id}`}
         pathPrefix={`product`}
+        isFlipImage={true}
```
Comment (inezamandha) [inline components/ProductsComponent/ProductRecomendation.tsx:88]: di isFlipImage kan saja aman tidak virgii?

File: components/ProductsComponent/ProductsList.tsx [hunk 2]
```diff
@@ -103,6 +103,7 @@ const ProductsList: FC<ProductsListType> = ({
           collectionSlug={categories}
           tagName={tagname || ""}
           filter={filterProduct}
+          isFlipImage={true}
```
Comment (inezamandha) [inline components/ProductsComponent/ProductsList.tsx:106]: sama~

File: components/ProductsComponent/ProductsWidget.tsx [hunk 3]
```diff
@@ -49,6 +49,7 @@ const ProductsWidget: FC<ProductsWidgetType> = ({
           isButton
           fullPath={`product/{id}`}
           pathPrefix={`product`}
+          isFlipImage={true}
```
Comment (inezamandha) [inline components/ProductsComponent/ProductsWidget.tsx:52]: sama~

**Review by inezamandha — APPROVED** [review]

File: components/ProductsComponent/ProductRecomendation.tsx (hunk 1 above)
Comment (falmesino) [reply components/ProductsComponent/ProductRecomendation.tsx:88]: legenda mengatakan kalo nilai-nya true itu kaga usah ditulis si true-nya.
coba dulu aja bener apa kaga jangan langsung diikutin aowkaowkao

**Review by falmesino — APPROVED** [review]

**Pull request merged**